deploy/host-agent/tests/test_host_agent_handler.py

```
@@ -30,21 +32,27 @@ def _make_store() -> st.OperationStore:
+class _FakeProbe:
+    """Fake HealthProbe that returns True without opening sockets."""
+
+    def wait(self, host: str, port: int, timeout_seconds: int) -> bool:
+        return True
+
+
+def _make_executor(subprocess_run: Any = None) -> OperationExecutor:
     config = {
         "compose_project": "minecraft-bedrock",
         "compose_file": "/opt/craftcontrol/docker-compose.yml",
         "bedrock_data": "/tmp/bedrock-test",
```

Ruff (0.16.2)

[error] Probable insecure usage of temporary file or directory: "/tmp/bedrock-test"

(S108)